Source/Core/Core/PowerPC/JitCommon/Jit_Util.cpp

```diff
@@ -460,27 +498,41 @@ bool EmuCodeBlock::WriteToConstAddress(int accessSize, OpArg arg, u32 address, B
 	}
 }
 
+#include "disasm.h"
```

Comment:
Includes should be at the top of the source file, not smack in the middle of code.